dotnet/src/Connectors/Connectors.Memory.Redis/RedisMemoryStore.cs

```diff
@@ -246,12 +246,19 @@ public async Task RemoveBatchAsync(string collectionName, IEnumerable<string> ke
             cancellationToken: cancellationToken).FirstOrDefaultAsync(cancellationToken: cancellationToken).ConfigureAwait(false);
     }
 
+    /// <summary>
+    /// Dispose the object.
+    /// </summary>
     public void Dispose()
     {
         this.Dispose(true);
         GC.SuppressFinalize(this);
     }
 
+    /// <summary>
+    /// Dispose the object.
+    /// </summary>
+    /// <param name="disposing">True if disposing managed resources.</param>
```

Comment:
Should this comment be consistent across all classes. There is a different description in a previous class.